### ISSUE TITLE
Support for HDFS files

### ISSUE THREAD

**Issue by mrunesson** [SvenskaSpel/cobra-policytool#3]
When I tag a Hive table, I want the same tag to be applies files the table is stored in.

This requires to:
* look up path for table in metastore.
* Add tag for the path in Atlas, when tagging.

**Comment by mrunesson** [comment]
Plan to handle tag files is to add a command line option --hdfs to tags_to_atlas command. This means  program will for each table defined in table_tags.csv look up its storage location and tag that storage location. 

Second step is extend command rules_to_ranger to be able to convert hive resources in ranger_policy.json to hdfs resources with similar access rules. This will only be done on commands in the ranger_policy.json file with block:
```
"options": {
        "expandHiveResourceToHdfs": true,
        "hdfsService": "hadoop"
}
```

For tag based rules, intention is to require user to extend the policy block to also cover hdfs.

Note: Row filtering and masking will still not work on jobs reading from HDFS.

**Comment by mrunesson** [comment]
Released in v1.1.0
